### PR TITLE
connect other cities api

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request we want to connect other cities API's

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,8 +12,13 @@
   </head>
   <body>
     <header class="header">
-      <i class="fa fa-bolt fa-magnifying-glass"></i>
-      <input class="header-search" type="text" placeholder="Search.." />
+      <!-- <i class="fa fa-bolt fa-magnifying-glass"></i>
+      <input class="header-search" type="text" placeholder="Search.." /> -->
```
Comment (blacksrc) [inline index.html]: Please, remove these.

File: assests/javascript/index.js [hunk 2]
```diff
@@ -17,9 +22,9 @@ async function getWeather() {
   }
 }
 
-async function getCityLocation() {
+async function getCityLocation(cityName) {
   const fetchedData = await fetch(
-    `${baseUrl}geo/1.0/direct?q=Rasht,IR&appid=${apiKey}`
+    `${baseUrl}geo/1.0/direct?q=${cityName}&appid=${apiKey}`
```
Comment (blacksrc) [inline assests/javascript/index.js:27]: Is this change correct? The previous value was `Rasht,IR` and now it's `rasht`. 
I recommend to use city id if it's available in the api (check the api and see if there is a unique identifier for the cities and if it's possible to fetch the cities by that)

File: index.html [hunk 3]
```diff
@@ -12,8 +12,13 @@
...
+      <select id="cities">
+        <option value="rasht" selected>rasht</option>
+        <option value="tehran">tehran</option>
+        <option value="stockholm">stockholm</option>
```
Comment (blacksrc) [inline index.html]:
```suggestion
        <option value="rasht" selected>Rasht</option>
        <option value="tehran">Tehran</option>
        <option value="stockholm">Stockholm</option>
```

File: index.html (hunk 1 above)
Comment (mim24sd) [reply index.html]: sure

File: index.html (hunk 3 above)
Comment (mim24sd) [reply index.html]: thanks

File: assests/javascript/index.js (hunk 2 above)
Comment (mim24sd) [reply assests/javascript/index.js:27]: you are right,I will fix it

**Pull request merged**